core/serialisation/src/main/java/uk/gov/gchq/gaffer/serialisation/implementation/MultiSerialiser.java

```diff
@@ -40,17 +40,18 @@
  *             ^
  *             serialiser byte key
  * </pre>
- * When multiple serialisers that operate on the same value type are added,
- * the last one to be added will be used for serialisation. This happens
- * regardless of the key value used when adding to the MultiSerialiser (shown in example)
+ * When multiple serialisers are used that operate on the same value class then
+ * the last serialiser in the list will be used for serialisation; this happens
+ * regardless of the key's natural-ordering. In the example Integers will be serialsed with no.8.
  * <br>
  * For deserialising the correct serialiser is always chosen based on the byte key at the start of the byte[].
  * <p>
```

Comment:
Appreciate that the code was already like this, but where's the end of this paragraph? While the Javadoc is ok with this, it would be better to use start and end `<p>` tags and not `<br>` (unless a newline is the intention).

Reply:
It's not consistent if `<p>` requires closing. I had seen some warning about it, but they were not for this file. So logn as the formatting is workign as intended it can stay as is.